server/src/github.com/alexeyknyshev/cpsrv/cpsrv.go

```diff
@@ -39,54 +38,36 @@ type Message struct {
 	Text string `json:"text"`
 }
 
-func checkConvertionUint(val uint32, err error, context string) uint32 {
-	if err != nil {
-		log.Printf("%s: uint conversion err => %v\n", context, err)
-		return 0
-	}
-	return val
+type HandlerContextStruct struct {
+	Tnt *tarantool.Connection
+	Logger
+}
+
+type HandlerContext interface {
+	tnt() *tarantool.Connection
+	logger() Logger
 }
 
-func logRequest(w http.ResponseWriter, r *http.Request, requestId int64, requestBody string) error {
-	endpointStr := r.URL.Path
-	if requestBody != "" {
-		endpointStr = endpointStr + " =>"
-		body := ""
+func (handler HandlerContextStruct) tnt() *tarantool.Connection {
+	return handler.Tnt
+}
 
-		//if isTestingModeEnabled(redisCli) {
-		//	prettyJson, err := jsonPrettify(requestBody)
-		//	if err == nil {
-		//		body = "\n" + prettyJson
-		//	} else {
-		//		body = " " + requestBody
-		//	}
-		//}
-		body = " " + requestBody
-		endpointStr = endpointStr + body
-	}
-	log.Printf("%s Request: %s %s", getRequestContexString(r), r.Method, endpointStr)
-	return nil
+func (handler HandlerContextStruct) logger() Logger {
+	return handler.Logger
 }
 
-func logResponse(w http.ResponseWriter, r *http.Request, requestId int64, responseBody string) error {
-	endpointStr := r.URL.Path
-	if responseBody != "" {
-		endpointStr = endpointStr + " =>"
-		body := ""
+func makeHandlerContext(tnt *tarantool.Connection) HandlerContext {
+	logger := &TestLogger{make(chan string)}
```

Comment:
Why don't create tnt inside this func?
It would allow us to call only this func to prepare working context